common/src/main/java/com/hedera/block/common/utils/FileUtilities.java

```diff
@@ -62,21 +65,22 @@ public final class FileUtilities {
                             PosixFilePermission.OTHERS_EXECUTE));
 
     /**
-     * Log message template used when a path is not created because a file
-     * or folder already exists at the requested path.
+     * Log message template used when a path is not created because a file or folder already exists
+     * at the requested path.
```

Comment:
Javadoc should wrap with shorter lines; it's intended to be readable in github which locks down to a narrow viewport for some incomprehensible reason.

Reply:
✅ I reversed the formatting on the docs everywhere.